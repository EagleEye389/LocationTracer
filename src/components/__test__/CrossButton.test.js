import React from "react";
import { shallow } from "enzyme";
import CrossButton from "../crossButton/CrossButton";

let component;
describe("CrossButton is ok", () => {
  beforeEach(() => {
    component = shallow(
      <CrossButton value={true} name="destination" onChangeInput={jest.fn()} />
    );
  });
  it("Rendering correctly", () => {
    expect(component).toMatchSnapshot();
  });
});
