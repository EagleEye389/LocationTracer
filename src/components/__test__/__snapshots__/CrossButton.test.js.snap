// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CrossButton is ok Rendering correctly 1`] = `ShallowWrapper {}`;
